BepInEx/Utility.cs

```diff
@@ -61,9 +81,8 @@ public static IEnumerable<TNode> TopologicalSort<TNode>(IEnumerable<TNode> nodes
 				Stack<TNode> currentStack = new Stack<TNode>();
 				if (!Visit(input, currentStack))
 				{
-					throw new Exception("Cyclic Dependency:\r\n" + currentStack
-																   .Select(x => $" - {x}") //append dashes
-																   .Aggregate((a, b) => $"{a}\r\n{b}")); //add new lines inbetween
+					throw new Exception("Cyclic Dependency:\r\n" + currentStack.Select(x => $" - {x}") //append dashes
+																			   .Aggregate((a, b) => $"{a}\r\n{b}")); //add new lines inbetween
```

Comment:
Note for myself: `String.Join()` probably would've been a better pick instead of `Aggregate()`